server/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java

```diff
@@ -5429,7 +5429,9 @@ public void testRefreshScopedSearcher() throws IOException {
                 engine.index(primaryResponse);
             }
             assertTrue(engine.refreshNeeded());
-            engine.refresh("test", Engine.SearcherScope.INTERNAL, true);
+            var refreshResult = engine.refresh("test", Engine.SearcherScope.INTERNAL, true);
+            assertTrue(refreshResult.refreshed());
+            assertNotEquals(refreshResult.generation(), Engine.RefreshResult.UNKNOWN_GENERATION);
```

Comment:
The `refresh` return value is ignored in the tests since there were no return values. This is the only place I've updated since it expects a refresh to happen.